dist/jowl.spec

```diff
@@ -0,0 +1,87 @@
+%global npm_name jowl
```

Comment:
Why are you using `%npm_name`? Couldn't you just use `%name`?

Reply:
Good point. I got that from a specfile that deliberately needed to use a different package name than the npm name but since they're the same here, I can just use `%name`.